src/components/can_cooperation/include/can_cooperation/commands/get_interior_vehicle_data_capabilities_request.h

```diff
@@ -0,0 +1,103 @@
+ disclaimer in the documentation and/or other materials provided with the
+ distribution.
+
+ Neither the name of the Ford Motor Company nor the names of its contributors
+ may be used to endorse or promote products derived from this software
+ without specific prior written permission.
+
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef SRC_COMPONENTS_CAN_COOPERATION_INCLUDE_CAN_COOPERATION_COMMANDS_GET_INTERIOR_VEHICLE_DATA_CAPABILITIES_REQUEST_H_
+#define SRC_COMPONENTS_CAN_COOPERATION_INCLUDE_CAN_COOPERATION_COMMANDS_GET_INTERIOR_VEHICLE_DATA_CAPABILITIES_REQUEST_H_
+
+#include "can_cooperation/commands/base_command_request.h"
+#include "can_cooperation/event_engine/event.h"
+#include "can_cooperation/can_module_interface.h"
+
+namespace can_cooperation {
+
+namespace commands {
+
+/**
+ * @brief GetInteriorVehicleDataCapabiliesRequest command class
+ */
+class GetInteriorVehicleDataCapabiliesRequest : public BaseCommandRequest {
+ public:
+  /**
+   * @brief GetInteriorVehicleDataCapabiliesRequest class constructor
+   *
+   * @param message Message from mobile
+   **/
+  explicit GetInteriorVehicleDataCapabiliesRequest(
+      const application_manager::MessagePtr& message,
+      CANModuleInterface& can_module);
+
+  /**
+   * @brief Execute command
+   */
+  virtual void Execute();
+
+  /**
+   * @brief executes specific message validation
+   */
+  virtual bool Validate();
+
+  /**
+   * @brief Interface method that is called whenever new event received
+   *
+   * @param event The received event
+   */
+  void OnEvent(const can_event_engine::Event<application_manager::MessagePtr,
+                                             std::string>& event);
+
+  /**
+   * @brief GetInteriorVehicleDataCapabiliesRequest class destructor
+   */
+  virtual ~GetInteriorVehicleDataCapabiliesRequest();
+
+ protected:
+  virtual application_manager::TypeAccess CheckAccess(
+      const Json::Value& message);
+  virtual std::string ModuleType(const Json::Value& message);
+
+ private:
+  void UpdateModules(Json::Value* params);
+  application_manager::TypeAccess CheckModuleTypes(const Json::Value& message);
+  bool ReadCapabilitiesFromFile();
+  void CreateCapabilities(const Json::Value& zone_capabilities,
```

Comment:
@AByzhynar 
Currently there is no clarification from customer on format expected for fallback capabilities, so I would leave these parts untill further information will be provided and than change that accordingly.